ios/sdk/src/callkit/JMCallKitProxy.swift

```diff
@@ -47,6 +46,16 @@ import Foundation
         }
     }
     
+    /// Enables the proxy in between callkit and the consumers of the SDK
+    /// Default to enabled, set to false when you don't want to use callkit
+    @objc public static var callKitProxyEnabled: Bool = true {
+        didSet {
+            if callKitProxyEnabled == false {
+                cxProvider.setDelegate(nil, queue: nil)
```

Comment:
This would crash in iOS 9, right? we still support it in the SDK. I know there is no callkit there, but we shouldn't crash if someone sets it to whatever value.

Reply:
you are right, I'll make the jmcallkitproxy interface only available for ios10